src/lib/viewers/image/ImageControls.tsx

```diff
@@ -1,18 +1,29 @@
 import React from 'react';
 import AnnotationsControls, { Props as AnnotationsControlsProps } from '../controls/annotations';
+import ColorPickerControl from '../controls/color-picker';
 import ControlsBar from '../controls/controls-bar';
 import FullscreenToggle, { Props as FullscreenToggleProps } from '../controls/fullscreen';
 import RotateControl, { Props as RotateControlProps } from '../controls/rotate';
 import ZoomControls, { Props as ZoomControlsProps } from '../controls/zoom';
+import { AnnotationColor } from '../../AnnotationModule';
+import { AnnotationMode } from '../controls/annotations/types';
 
-export type Props = AnnotationsControlsProps & FullscreenToggleProps & RotateControlProps & ZoomControlsProps;
+const colors = Object.values(AnnotationColor);
```

Comment:
Do we need to do this in each set of controls? Should this array be exported from `AnnotationModule`?

Reply:
Yes it can live there instead.